### PR TITLE
add support for Diameters leader lines insode or outside a radius; make Diameter dimension value mobile

### PR DESCRIPTION
**Pull request by exegetor**

### CODE REVIEW

File: class_defines.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from distutils import core
```
Comment (hlorus) [inline class_defines.py:1]: This looks like a leftover import

**Review by hlorus — COMMENTED** [review]
Generally this looks fine. The draw_inside properties of the distance and angle constraints are probably WIP. The patch could already be merged if they are removed from it, however if you prefer you can also just update whenever those changes are done as well.

File: class_defines.py (hunk 1 above)
Comment (exegetor) [reply class_defines.py]: good observations, thanks for the review.   PR updated to fix.

**Comment by hlorus** [comment]
Alright this is already a great improvement and can go in like that. I'm just pointing a few things out which will be great additions but can come in later:
- Improving the default placement position to be on the outside
- Changing the operator on the text to the same as the line so it can be dragged from the text as well.

**Pull request merged**